tests/model_specific/test_llama_cpp.py

```diff
@@ -49,8 +51,11 @@ def test_llama_cpp_select2(llamacpp_model: guidance.models.Model):
     ]
 
 
+@pytest.mark.skipif(
```

Comment:
Maybe xfailing (non-strict) would be better so we can continue monitoring the state of these tests?

Reply:
Ooohhh, I hadn't noticed that `xfail` had a `condition` option. Agree that that's better.